src/types/ast.ts

```diff
@@ -174,7 +179,7 @@ export interface BinaryExpression extends BaseNode {
 export interface CallExpression extends BaseNode {
   callee: Expression
   arguments?: Expression[]
-  type: 'CallExpression'
+  type?: 'CallExpression' // observed in ASTs. type not always present.
```

Comment:
I am hesitant about this.
Found for range expressions. The rest of the node had the proper structure...but it was not typed.